functions-app/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "functions-app",
+  "version": "0.0.1",
+  "description": "MTC Azure Functions App Service Plan",
```

Comment:
Please can we add the purpose in here instead - that they are long runnng functions

Reply:
Done